client/settings/express-checkout/index.js

```diff
@@ -77,7 +77,10 @@ const ExpressCheckout = () => {
 			<CardBody size={ 0 }>
 				<ul className="express-checkouts-list">
 					{ isPlatformCheckoutFeatureFlagEnabled && (
-						<li className="express-checkout">
+						<li
+							className="express-checkout"
+							id="express-checkouts-woopay"
```

Comment:
Nitpicky, but should this be `express-checkout__woopay`?  The class names aren't pluralized and we can use BEM style selector names like we do in other components.